packages/pods-core/src/builtin/GDocPod.ts

```diff
@@ -0,0 +1,273 @@
+import { ImportPod, ImportPodConfig, ImportPodPlantOpts } from "../basev3";
+import { JSONSchemaType } from "ajv";
+import { PodUtils } from "../utils";
+import axios from "axios";
+import { googleDocsToMarkdown } from "docs-markdown";
+import _ from "lodash";
+import {
+  DendronError,
+  DVault,
+  NoteProps,
+  NoteUtils,
+  stringifyError,
+  DEngineClient,
+} from "@dendronhq/common-all";
+
+const ID = "dendron.gdoc";
+
+type GDocImportPodCustomOpts = {
+  /**
+   * google docs personal access token
+   */
+  token: string;
+
+   /**
+   * document Id of doc to import
+   */
+  documentId: string;
```

Comment:
for my understanding: What's the right documentId to specify here? Is it something like `meet.planning.triage` or is something like `1t9IoUjKs9sDa6vR3BvwXX1leABiTnfT-f1aPupBgpyI` (as seen from the URL when viewing the gdoc https://docs.google.com/document/d/1t9IoUjKs9sDa6vR3BvwXX1leABiTnfT-f1aPupBgpyI/edit)

Maybe we should add some additional notes either here and/or on the dendron-site wiki so that it's clearer what should be used here.

Reply:
The document id should be `1t9IoUjKs9sDa6vR3BvwXX1leABiTnfT-f1aPupBgpyI` from the url of doc. I should update about this field's specifications in the documentation

Reply:
The document id should be 1t9IoUjKs9sDa6vR3BvwXX1leABiTnfT-f1aPupBgpyI from the url of doc. I should update about this field's specifications in the documentation